datafusion/datasource/src/statistics.rs

```diff
@@ -230,14 +230,7 @@ impl MinMaxStatistics {
                 .zip(sort_columns.iter().copied())
                 .map(|(sort_expr, column)| {
                     let schema = values.schema();
-
```

Comment:
This appears to be the only actual code change: remove these lines

`git praise` says they came in via  #9593 from @suremarc . Do you remember why this condition was added @suremarc ?

https://github.com/apache/datafusion/blame/a614716e7d97ff1d3374aef31b9a66fd10141423/datafusion/datasource/src/statistics.rs#L238